packages/jsii-pacmak/generate.sh

```diff
@@ -2,9 +2,11 @@
 set -euo pipefail
 
 # embed the dotnet code generator program as a resource
-dng="lib/targets/dotnet-generator"
-mkdir -p ${dng}
-rsync -av node_modules/jsii-dotnet-generator/cli/publish/ ${dng}/
+if ! ${NO_DOTNET:-false}; then
```

Comment:
What about Java/Python?

Reply:
The generators for those are written in TypeScript, so they can be compiled (just not used to compile Java/Python packages).

This is about compiling the jsii packages themselves.